app/bundles/LeadBundle/Model/ImportModel.php

```diff
@@ -150,22 +150,45 @@ public function generateLink(Import $import)
     }
 
     /**
-     * Check if there are some IN_PROGRESS imports which got stuck for a while.
-     * Set those as failed.
+     * Returns import max runtime status from the configuration
+     * when max runtime has been exceeded.
+     *
+     * @param int $default
+     *
+     * @return int
      */
-    public function setGhostImportsAsFailed()
```

Comment:
Changing public methods on a public service is a BC break. Can you provide proxy methods to the new method so that setGhostImportsAsFailed and getGhostImportsAsFailed continue to function?